chalice_spec/chalice.py

```diff
@@ -167,8 +184,9 @@ def route(self, path: str, **kwargs: Any) -> Callable[..., Any]:
         def route_decorator(func):
             docs: Docs = kwargs.pop("docs", None)
             methods = [method.lower() for method in kwargs.get("methods", ["get"])]
+            content_types = kwargs.pop("content_types", None)
```

Comment:
Since you would want `content_types` to pass through to the underlying `route` call, should this be `kwargs.get` instead of `kwargs.pop`?

Reply:
Nice catch, thanks! Fixed in 698d447b3cf7ff46dfe95e5a9ef30a393482ede1